libs/s25main/lua/LuaInterfaceSettings.h

```diff
@@ -36,6 +36,7 @@ class LuaInterfaceSettings : public LuaInterfaceGameBase
     /// Get addons that are allowed to be changed
     std::vector<AddonId> GetAllowedAddons();
     bool IsMapPreviewEnabled();
+    unsigned GetNumPlayersFromScript();
```

Comment:
```suggestion
    /// Get number of players as set in the script. Return zero to use the value set in the map/save file.
    unsigned GetNumPlayersFromScript();
```

Or maybe returning an optional is more clear and disallow zero for lua files.